<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SURVEC v1.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/resources/icons/Icone.png</normaloff>:/resources/resources/icons/Icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>925</width>
         <height>516</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="cwROIResult" name="cw_ROIResult1" native="true"/>
            </item>
            <item row="1" column="0">
             <widget class="cwROIResult" name="cw_ROIResult2" native="true"/>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="4" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>88</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0">
               <widget class="QGroupBox" name="gbVideoRecording">
                <property name="title">
                 <string>Video Recording</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="cbRecord1">
                   <property name="text">
                    <string>ROI 1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="cbRecord2">
                   <property name="text">
                    <string>ROI 2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="lbMaxDuration">
                   <property name="text">
                    <string>Maximum duration:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDateTimeEdit" name="dteMaxDuration">
                   <property name="keyboardTracking">
                    <bool>true</bool>
                   </property>
                   <property name="time">
                    <time>
                     <hour>1</hour>
                     <minute>0</minute>
                     <second>0</second>
                    </time>
                   </property>
                   <property name="minimumTime">
                    <time>
                     <hour>0</hour>
                     <minute>0</minute>
                     <second>2</second>
                    </time>
                   </property>
                   <property name="displayFormat">
                    <string>HH:mm:ss</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupBoxJournal">
                <property name="title">
                 <string>Alarm log</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkBoxWriteResults">
                   <property name="text">
                    <string>Write results</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QCheckBox" name="checkBoxWriteImages">
                   <property name="text">
                    <string>Write Images</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QGroupBox" name="outputInterface">
                <property name="title">
                 <string>Output Interface</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="comboBoxOutput">
                   <property name="toolTip">
                    <string>The output interface are defined as Survec plugins. You need to install plugin to populate this list.</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QGroupBox" name="gbResultOp">
                <property name="title">
                 <string>Logical Combination</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="radioButtonOR">
                   <property name="text">
                    <string>OR</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="radioButtonAND">
                   <property name="text">
                    <string>AND</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item alignment="Qt::AlignLeft">
               <widget class="QToolButton" name="tbStartProcess">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Start process of alarm detection</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/resources/resources/icons/startProcess.png</normaloff>:/resources/resources/icons/startProcess.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>150</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonIconOnly</enum>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="lbTime">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>00:00:00</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="lbMainAlarm">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Alarm state (from the combination of one or two ROI alarms)</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resources.qrc">:/resources/resources/icons/ledNO.png</pixmap>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lbVideoRecording">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>120</red>
                      <green>120</green>
                      <blue>120</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>120</red>
                      <green>120</green>
                      <blue>120</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>lbVideoRecording</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="lbAvgTime">
                <property name="text">
                 <string>lbAvgTime</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lbFPS">
                <property name="text">
                 <string>lbFPS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="gridLayoutWidget">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QGroupBox" name="gbVideoROI1">
               <property name="title">
                <string>ROI 1</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="cwVideoPlayer" name="videoPlayer1" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="gbVideoROI2">
               <property name="title">
                <string>ROI 2</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <widget class="cwVideoPlayer" name="videoPlayer2" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QMenu" name="menuLanguages">
     <property name="title">
      <string>Languages</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionFrench"/>
    </widget>
    <addaction name="actionEditColors"/>
    <addaction name="separator"/>
    <addaction name="menuLanguages"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwAlarmLog">
   <property name="windowTitle">
    <string>Alarm log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pbResetHisto">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbOpenDir">
         <property name="text">
          <string>Open dir</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QTableWidget" name="tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Duration (s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Alarm Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type ROI 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value ROI 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Target ROI 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Logic</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type ROI 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value ROI 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Target ROI 2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionEditColors"/>
   <addaction name="actionAlarm"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionInfo"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/Filenew.png</normaloff>:/resources/resources/icons/Filenew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/Fileopen.png</normaloff>:/resources/resources/icons/Fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/Filesave.png</normaloff>:/resources/resources/icons/Filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/Filesaveas.png</normaloff>:/resources/resources/icons/Filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionEditColors">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/coloredit.png</normaloff>:/resources/resources/icons/coloredit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Colors</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/info.png</normaloff>:/resources/resources/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/help.png</normaloff>:/resources/resources/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/power.png</normaloff>:/resources/resources/icons/power.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/englishflag.png</normaloff>:/resources/resources/icons/englishflag.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/frenchflag.png</normaloff>:/resources/resources/icons/frenchflag.png</iconset>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionAlarm">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/resources/icons/Icon-Archive.png</normaloff>:/resources/resources/icons/Icon-Archive.png</iconset>
   </property>
   <property name="text">
    <string>Alarm</string>
   </property>
   <property name="toolTip">
    <string>Alarm log</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cwVideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">cwvideoplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cwROIResult</class>
   <extends>QWidget</extends>
   <header location="global">cwroiresult.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
